langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -188,9 +188,9 @@ function testLastIndexOf() {
     }
 }
 
-function testReverse() returns [int[], int[]] {
+function testReverse() returns [int[]] {
     int[] arr = [10, 20, 30, 40, 50];
-    return [arr, arr.reverse()];
+    return [arr.reverse()];
```

Comment:
We can simply pass the array as return type instead of a tuple here. Shall we change it?
```suggestion
    return arr.reverse();
```